include/zephyr/net/wifi_nm.h

```diff
@@ -94,6 +94,10 @@ int wifi_nm_unregister_mgd_iface(struct wifi_nm_instance *nm, struct net_if *ifa
  * @}
  */
 
+#define SSID_MAX_LEN 32
+#define ETH_ALEN 6
```

Comment:
This would be more useful in `ethernet.h` file. We seem to also have several MAC address length set in various header files and those could be replaced by `ETH_ALEN`, but that can be done in separate PR in the future.

Reply:
Will take care of this in a separate PR.